server/server.go

```diff
@@ -152,6 +152,7 @@ func (f *FallbackServer) dial() (connection, error) {
 
 // options is a handler for the OPTIONS call that precedes CORS-enabled calls.
 func (f *FallbackServer) options(w http.ResponseWriter, r *http.Request) {
+	log.Println("Incoming OPTIONS for request:", r.RequestURI)
```

Comment:
This is not an error condition and will spam in the log for each `OPTIONS` request - is it intended?

Reply:
Ah sorry I merged this right before you commented! Yes this is intended, I figure more logging on request handling is fine. This shouldn't handle a lot of requests and I'd prefer having the extra logs to follow request execution. If this were to be "productionized" then this would be extraneous.